src/main/java/seedu/address/commons/util/DateTimeUtil.java

```diff
@@ -2,16 +2,18 @@
 
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.ResolverStyle;
 
 public class DateTimeUtil {
 
     /**
-     * Returns a case-insensitive DateTimeFormatter according to the given pattern
+     * Returns a case-insensitive DateTimeFormatter according to the given pattern with a Strict resolver style
```

Comment:
maybe can add a test showcasing the strict nature as well?

Reply:
i lazy

Reply:
modbookdate already has tests for the strict resolver style, so i just added a test for modbooktime